texthero/representation.py

```diff
@@ -14,6 +14,13 @@
 from scipy.sparse import coo_matrix
 
 from typing import Optional, Union, Any
+from texthero._types import (
+    TextSeries,
+    TokenSeries,
+    VectorSeries,
+    DataFrame,
+    InputSeries,
+)
 
 from texthero import preprocessing
```

Comment:
Can we remove this import as mentioned in #128?

Reply:
I would not remove this here as we're trying our best to keep the pull requests atomic. I think it's best to do this in a separate PR

Reply:
Sure thing! looking forward to your merge.